lib/bolt/executor.rb

```diff
@@ -127,6 +146,13 @@ def report_function_call(function)
       @analytics&.event('Plan', 'call_function', function)
     end
 
+    def report_bundled_content(mode, name)
+      if BUNDLED_CONTENT.include?(name)
+        screen = "bundled_#{mode}_content"
+        @analytics&.screen_view(screen, bundled_content: name)
```

Comment:
This makes more sense as an event, similar to calling a function or using a transport. Could be something like `event('Bundled Content', mode, name)`.